cime_config/allactive/config_compsets.xml

```diff
@@ -311,11 +311,27 @@
   <lname>SSP585SOI_EAM%CMIP6_ELM%CNPECACNTBC_MPASSI_MPASO_MOSART_SGLC_SWAV_BGC%BDRC</lname>
 </compset>
 
-<compset>
+<ompset>
   <alias>BGCEXP_BDRD_CNPECACNT_20TR</alias>
   <lname>20TR_EAM%CMIP6_ELM%CNPECACNTBC_MPASSI%BGC_MPASO%OIECOOIDMS_MOSART_SGLC_SWAV_BGC%BDRD</lname>
 </compset>
 
+<compset>
+  <alias>BGCEXP_BDRD_CNPECACNT_SSP585_CMIP6</alias>
+  <lname>SSP585_EAM%CMIP6_ELM%CNPECACNTBC_MPASSI%BGC_MPASO%OIECOOIDMS_MOSART_SGLC_SWAV_BGC%BDRD</lname>
+</compset>
+
+<compset>
+  <alias>BGCEXP_BDRD_CNPECACNT_20TRS</alias>
```

Comment:
I noticed that here (and elsewhere) we are retaining compsets both with and without spunup initial conditions (indicated by the "S" at the end of the compset alias).  If this is the current practice of E3SM and is also used by the water cycle compsets (after their recent cleanup), then I think we should follow their example and keep both versions.  Can you confirm whether the water cycle compsets are keeping both versions (with and without "S") for each compset?

Reply:
@susburrows 
The master that I checked out still kept using 'S' at the end of the compost alias for the other BGC composts. I just switched the BCRC to CNTL for the control simulation composts and added three more composts for ECA BDRD.

 I followed the convention of the water cycle composts. Please check the below water cycle composts:

<compset>
<alias>A_WCYCL20TR</alias>
<lname>20TR_EAM%AV1C-L_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
</compset>

<compset>
<alias>A_WCYCL20TRS</alias>
<lname>20TRSOI_EAM%AV1C-L_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
</compset>

But I did not know what is SOI mean? who do you think I should ask for this point?

Reply:
A_WCYCL20TR 20TR.  
20TR_EAM%AV1C-L_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV

A_WCYCL20TRS. 
20TRSOI_EAM%AV1C-L_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV

Reply:
Hi Xiaoying, thanks for clarifying those points.  I think that we are good to go with this version in that case.